### PR TITLE
Implement #1112

### PR DESCRIPTION
**Pull request by Felixmil**
For #1112

NEW:
- feature that add scaling for residuals (lin or log)
- unit tests

UPDATED:
- Documentation 
- Vignette

I didn't know if I had to render the whole pkgdown website, in double, I just re-rendered the impacted vignette and commited the `.html` file and new `.png` only.

### CODE REVIEW

File: R/plot-residuals-vs-simulated.R [hunk 1]
```diff
@@ -43,20 +44,25 @@
 #' myPlotConfiguration$caption <- "My Sources"
 #'
 #' # plot
-#' plotResidualsVsSimulated(myDataCombined, myPlotConfiguration)
+#' plotResidualsVsSimulated(myDataCombined, scaling = "log", defaultPlotConfiguration = myPlotConfiguration)
 #'
 #' @export
 plotResidualsVsSimulated <- function(dataCombined,
-                                     defaultPlotConfiguration = NULL) {
+                                     defaultPlotConfiguration = NULL,
```
Comment (Yuri05) [inline R/plot-residuals-vs-simulated.R:51]: why is it actually called **default**PlotConfiguration and not just plotConfiguration?

Comment (PavelBal) [reply R/plot-residuals-vs-simulated.R:51]: Because `PlotConfiguration` is already used in `TLF`...

**Comment by PavelBal** [comment]
@Felixmil You must run the tests locally and accept the changed snapshot (if they should be accepted...) and then push the updated snapshots, otherwise, the built will fail.

File: R/plot-residuals-vs-simulated.R (hunk 1 above)
Comment (Yuri05) [reply R/plot-residuals-vs-simulated.R:51]: ?? it's a function argument, you can call it whatever you want.
```

plotResidualsVsSimulated <- function(dataCombined,
                                     plotConfiguration = NULL,
                                     scaling="lin") {
```

Not a big deal, we can keep it AS IS. Just wondering.

Comment (PavelBal) [reply R/plot-residuals-vs-simulated.R:51]: Ah no I understand, you mean the argument name. I prefer to have the class name so it is clear why type is required.

**Comment by PavelBal** [comment]
@Yuri05 @msevestre What kind of sorcery is this? The latest TLF version from the `develop` branch is 1.5.124:

image

However when installing the package as is done in the AppVeyor script:

`install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip', repos = NULL, type = 'binary')`

the installed version is 1.5.125:

image

HOW??? What is this 1.5.125 build?? With this version, plots look different and this is why this PR fails.

**Review by PavelBal — APPROVED** [review]

**Comment by Yuri05** [comment]
we have one open PR on the tlf and it seems that the AppVeyor permalink takes the artifact from the PR.
This should not be the case!
After checking the [AppVeyor docu](https://www.appveyor.com/docs/packaging-artifacts/#permalink-to-the-last-successful-build-artifact) on this topic: it is possible to exclude the PRs by adding `?pr=false` to the permalink.

grafik

will create issues for this in all R repos

grafik

**Pull request merged**